docs/administration-guide/gaffer-stores/store-guide.md

```diff
@@ -9,6 +9,35 @@ Gaffer currently supplies the following store implementations:
 - [Proxy Store](proxy-store.md) - Delegates/forwards queries to another Gaffer REST
 - [Federated Store](federated-store.md) - Federates queries across multiple graphs
 
+## Store Properties
+
+Stores are configured using `key=value` style properties stored in a `store.properties` file.
+There are general properties which apply to all Stores and per Store properties for configuring specific behaviour.
+Most properties are optional and don't need to be specified or configured, default values will be used.
+
+### All General Store Properties
+
+The properties in bold are set based on the type of Gaffer Store, for how to configure these see the [respective page for each store type](#stores-guide).
+
+| Property                 | Default | Description                             |
+| ------------------------ | ------- | --------------------------------------- |
+| **`gaffer.store.class`** | N/A | Class Name String to set Gaffer Store class |
+| `gaffer.store.schema.class` | `gaffer.store.schema.Schema` | Class Name String to set class to use for serialising Schemas |
+| **`gaffer.store.properties.class`** | `uk.gov.gchq.gaffer.store.StoreProperties` | Class Name String to set Gaffer Store Properties class |
+| `gaffer.store.operation.declarations` | N/A | Path to [Operation Declarations](../../development-guide/example-deployment/project-setup.md#operations-declarations) files (separate multiple files with commas) |
```

Comment:
Maybe these could say Default: None? Rather than not applicable

Reply:
Just the ones which are required or everything which is `N/A`? I was using it to mean there was no default, what you suggest is probably clearer.

Reply:
Hmm, I was thinking everywhere None makes more sense than N/A. But it might also be worth stating which are required as well? (None: Required) perhaps?